src/Parser/Shaman/Elemental/Modules/Talents/LiquidMagmaTotem.js

```diff
@@ -0,0 +1,47 @@
+import React from 'react';
+
+import SPELLS from 'common/SPELLS';
+import SpellIcon from 'common/SpellIcon';
+import { formatNumber, formatPercentage } from 'common/format';
+
+import Analyzer from 'Parser/Core/Analyzer';
+
+import StatisticBox, { STATISTIC_ORDER } from 'Interface/Others/StatisticBox';
+
+class LiquidMagmaTotem extends Analyzer {
+  damageGained = 0;
+
+  constructor(...args) {
+    super(...args);
+    this.active = this.selectedCombatant.hasTalent(SPELLS.LIQUID_MAGMA_TOTEM_TALENT.id);
+  }
+
+  on_damage(event) {
+    if (event.ability.guid !== SPELLS.LIQUID_MAGMA_TOTEM_DAMAGE.id) {
+      return;
+    }
+    this.damageGained += event.amount;
+  }
+
+  get damagePercent() {
+    return this.owner.getPercentageOfTotalDamageDone(this.damageGained);
+  }
+
+  get damagePerSecond() {
+    return this.damageGained / (this.owner.fightDuration / 1000);
+  }
+
+  statistic() {
+    return (
+      <StatisticBox
+        icon={<SpellIcon id={SPELLS.LIQUID_MAGMA_TOTEM_TALENT.id} />}
+        value={`~ ${formatPercentage(this.damagePercent)} %`}
+        label="Of total damage"
+        tooltip={`LMT contributed ${formatNumber(this.damagePerSecond)} DPS (${formatNumber(this.damageGained)} total damage).`}
```

Comment:
I'd recommend not using abbreviations in tooltips, there should be plenty of space to write out the full name.

Reply:
Sounds fair. Will fix asap